README.md

```diff
@@ -125,5 +141,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+Schema, like the word says is it like a scheme for a data base, it is a collection f objects for the data base.
+
+One to many is a relationship between to things in which one (a) has a lot of (b), but (b) has only one (a)!
 ```
```

Comment:
Looking for a little more here. A schema defines what a table will look like. In particular, it indicates what columns it will have, what data types they will contain and what constraints (e.g., `NOT NULL`, `UNIQUE`) they might have.

The purpose of the schema is not to add rows to a table.